addons/l10n_in_hr_holidays/tests/test_sandwich_leave.py

```diff
@@ -0,0 +1,51 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from freezegun import freeze_time
+from odoo.addons.l10n_in_hr_payroll.tests.common import TestPayrollCommon
```

Comment:
This will crash if payroll is not installed + the test is not adapted at all to last modifications.